src/Store/VimStoreConnector.re

```diff
@@ -1032,6 +1073,18 @@ let start =
         state,
         synchronizeEditorEffect(state),
       )
+    | Command("workbench.action.zoomIn") => (
+        state,
+        zoomEffect(state, ZoomIn),
+      )
+    | Command("workbench.action.zoomOut") => (
+        state,
+        zoomEffect(state, ZoomOut),
+      )
+    | Command("workbench.action.zoomReset") => (
+        state,
+        zoomEffect(state, ZoomReset),
+      )
```

Comment:
I don't see any reason for these and the effect to need to live in `VimStoreConnector`. Could you move it to `CommandStoreConnector` instead (unless there's a reason I don't see)?